doc/conftest.py

```diff
@@ -43,6 +43,14 @@ def setup_working_with_text_data():
         raise SkipTest("Skipping dataset loading doctests")
 
 
+def setup_loading_other_datasets():
+    try:
+        import pandas  # noqa
+    except ImportError:
+        raise SkipTest("Skipping loading_other_datasets.rst, "
+                       "pandas not installed")
```

Comment:
Nit:

```suggestion
        raise SkipTest("Skipping loading_other_datasets.rst "
                       "because Pandas is not installed.")
```

Reply:
Thanks @alfaro96, I tend to agree with you... but I'd rather not to make this message different from the others skipping pandas... unless we should change them all... :)

Reply:
I prefer consistency in the messages rather than paraphrasing.

LGTM then :)